### ISSUE TITLE
data normalization

### ISSUE THREAD

**Issue by AmirAflak** [moelgendy/grokking_deep_learning_for_computer_vision#50]
Hi,
In normalization part of [this notebook ](https://github.com/moelgendy/deep_learning_for_vision_systems/blob/master/chapter_04/chapter_4_improve_cifar_90_percent.ipynb) why do we apply std of train set to test set ? could it be seperated ? 
![Screenshot from 2023-03-27 17-39-25](https://user-images.githubusercontent.com/84932095/227965491-db0cadd1-e896-4487-8295-1be4af3ea9e4.png)

**Comment by ngvananh2508** [comment]
Because we assume that both testing and training sets derived from the same distribution so the std of testing and training sets are the same.
